src/content/docs/en/core-concepts/routing.mdx

```diff
@@ -219,6 +220,79 @@ const { title, text } = page;
 </html>
 ```
 
+## Redirects
+
+Sometimes you will need to redirect your readers to a new page, either permanently because your site structure has changed, or in response to an action such as logging in to an authenticated route.
+
+You can define rules to [redirect users to permanently-moved pages](#configured-redirects-experimental) in your Astro config. Or, [redirect users dynamically](#dynamic-redirects) as they use your site.
+
+### Configured Redirects (experimental)
+<Since v="2.6.0" />
+
+After enabling the experimental flag `redirects: true`, you can specify a mapping of permanent redirects in your Astro config. For most redirects, this is a mapping of an old route to the new route:
+
+```js title="astro.config.mjs" {5, 7-9}
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  experimental: {
+    redirects: true
+  },
+  redirects: {
+    '/old-page': '/new-page'
+  }
+});
+```
+
+These redirects follow the same rules as file-based routes. Dynamic routes are allowed as long as both the new and old route contain the same parameters, for example:
```

Comment:
Not sure how this works in English, but "new and old" is an implicit plural, and supposedly "route" should be changed to agree in number.
```suggestion
These redirects follow the same rules as file-based routes. Dynamic routes are allowed as long as both the new and old routes contain the same parameters, for example:
```

Reply:
yes, this absolutely correct, Yan!